app/components/Players.jsx

```diff
@@ -21,4 +21,27 @@ const Players = ({ numOfPlayers, currCards, decks }) => {
   )
 }
 
-export default Players;
\ No newline at end of file
+// alternative solution
+// it is more in keeping with the react communities style to try to use
+// map or other array methods that return values
+// React is heavily inspired by functional programming
+// which prefers these sorts of array operations.
+// There's nothing wrong with your solution
+// But it is good to see how many developers prefer to do this sort of thing when using react
+/*
+const Players = ({ numOfPlayers, currCards, decks }) => (
+  <div id="decks">
+    {Array.from('x'.repeat(numOfPlayers)).map((item, counter) => (
+            <div data-testid="player" id="player" key={counter + 1}>
+            <div data-testid="curr-card" className="currentCard">{currCards[counter]}</div>
+            <div>Player {counter + 1}</div>
+            <div data-testid="num-of-cards">Number of Cards: {decks[counter].length}</div>
+          </div>
+    ))}
+  </div>
+)
+*/
+
+export default Players;
+
+
```

Comment:
I like how clean your solution is - much more readable. Which resources do you recommend for writing code that matches the react community style?

Reply:
There's too many blogs to count. 
For this particular convention, read this:
https://reactjs.org/docs/lists-and-keys.html#embedding-map-in-jsx